pkg/controller/provider/ova-setup.go

```diff
@@ -262,3 +264,15 @@ func (r *Reconciler) isEnforcedRestrictionNamespace(namespaceName string) bool {
 
 	return enforceExists && enforceLabel == "restricted" && !(auditExists && auditLabel == "restricted")
 }
+
+func generateRandomSuffix() (randomSeed string) {
```

Comment:
+ should  add the random suffix for deployment and the service as well?

Reply:
nope